Bottom: 5fdacc00732756bfb3c9eb0ac52e13ed8fb0fc91
Top:    bb5591fc017dd7f4874df5fcebeccaeba96206b4
Author: scott snyder <sss@karma>
Date:   2020-02-03 00:24:41 -0500

cmake LibDeps workaround



---

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 504edf4..1658b65 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -149,5 +149,6 @@ DISPLAY_STD_VARIABLES()
 
 
 # generate and install following configuration files
-GENERATE_PACKAGE_CONFIGURATION_FILES( CEDConfig.cmake CEDConfigVersion.cmake CEDLibDeps.cmake )
+GENERATE_PACKAGE_CONFIGURATION_FILES( CEDConfig.cmake CEDConfigVersion.cmake  )
 
+INSTALL (FILES ${PROJECT_SOURCE_DIR}/cmake/CEDLibDeps.cmake DESTINATION lib/cmake )
diff --git a/cmake/CEDLibDeps.cmake b/cmake/CEDLibDeps.cmake
new file mode 100644
index 0000000..e69de29
